site/content/docs/user/ingress.md

```diff
@@ -70,6 +70,10 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main
 Now the Ingress is all setup. Wait until is ready to process requests running:
 
 {{< codeFromInline lang="bash" >}}
+kubectl wait --namespace ingress-nginx \
+  --for=condition=complete job/ingress-nginx-admission-patch \
+  --timeout=30s \
```

Comment:
i would add the same timeout as the other one

Reply:
why two waits? is not wait enough since one depend on the other?